### PR TITLE
buttons & inputs styles

### PR DESCRIPTION
**Pull request by vzhabchyk**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1,19 +1,108 @@
...
+  background: url(../images/background.png);
 }
-.secondary-heading {
-  font-size: 36px;
-  line-height: 43px;
+.header-btn-login {
+  width: 100px;
+  height: 40px;
+  background: transparent;
+  border: 1px solid #ffffff;
```
Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  border: 1px solid #fff;
```

File: css/main.css [hunk 2]
```diff
@@ -1,19 +1,108 @@
...
+  border-radius: 6px;
+  color: #ffffff;
```
Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  color: #fff;
```

File: css/main.css [hunk 3]
```diff
@@ -1,19 +1,108 @@
...
+  border-radius: 6px;
+  font-size: 16px;
+  margin: 15px;
 }
-.text {
+.user-first-name-input::placeholder,
+.user-last-name-input::placeholder,
+.user-phone-input::placeholder {
+  padding: 0 10px 0 30px;
+}
```
Comment (Plakhotenko) [inline css/main.css]: It should be one component.

File: index.html [hunk 4]
```diff
@@ -141,27 +143,25 @@ <h2>Contact</h2>
         explorer of the truth, the master-builder of human happiness.
       </p>
       <form>
-        <label>
-          <input name="user_first_name" type="text" placeholder="First Name">
-        </label>
-        <label>
+        <aria-label><input name="user_first_name" type="text" placeholder="First Name">
```
Comment (Plakhotenko) [inline index.html]: There is no tag aria-label, but there is an attribute.

**Review by Plakhotenko — CHANGES_REQUESTED** [review]

File: css/main.css [hunk 5]
```diff
@@ -1,19 +1,439 @@
+*,
+*:before,
+*:after {
+  box-sizing: border-box;
+}
 body {
-  font-family: 'Lato', sans-serif;
+  margin: 0;
+  font-family: 'Lato';
+  text-align: justify;
```
Comment (Plakhotenko) [inline css/main.css]: Bad idea. Please specify text-align: justify only for separate components. There is no need to set this property globally.

File: css/main.css [hunk 6]
```diff
@@ -1,19 +1,439 @@
...
+  cursor: pointer;
+  background: none;
+  border: none;
+}
+.conteiner {
+max-width: 1140px;
+padding-right: 15px;
+padding-left: 15px;
+margin-left: auto;
+margin-right: auto;
```
Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  max-width: 1140px;
  padding-right: 15px;
  padding-left: 15px;
  margin-left: auto;
  margin-right: auto;
```

File: css/main.css [hunk 7]
```diff
@@ -1,19 +1,439 @@
...
+  display: block;
+  clear: both;
+}
+.col {
+  float: left;
+  padding-left: 15px;
+  padding-right: 15px;
+}
+.col-1 {
+width: 8.33333%;
```
Comment (Plakhotenko) [inline css/main.css]: Check indents it the code.

Comment (Plakhotenko) [inline css/main.css]:
```suggestion
  width: 8.33333%;
```

File: css/main.css [hunk 8]
```diff
@@ -1,19 +1,439 @@
...
+  margin-right: 10px;
+}
+.header-nav {
+  text-align: center;
+}
+.header-nav_ul {
+  margin: 0;
+  padding: 0;
+}
+.header-nav_li {
```
Comment (Plakhotenko) [inline css/main.css]:
```suggestion
.header-nav-list-item {
```

File: css/main.css (hunk 5 above)
Comment (vzhabchyk) [reply css/main.css]: thanks for the comments, I will correct all mistakes!

**Pull request merged**